clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -100,7 +100,7 @@ object ElasticsearchIndex extends Elastic4sAutoDerivation {
     implicitly[ElasticsearchIndex[Document]]
 
   implicit val WgsUbam: ElasticsearchIndex[DocumentWgsUbam] =
-    ElasticsearchIndex(ElasticsearchFieldMapper.NumericBooleanDateAndKeywordFields)
+    ElasticsearchIndex(ElasticsearchFieldMapper.StringsToTextFieldsWithSubKeywords)
```

Comment:
With this change, do we still need two different mappers? I thought we added a 2nd because we needed different behaviors for different indexes. If all indexes use the same mapper I think we can fold them into a single class.

Reply:
It looks like there's probably some sneaky dependencies, but I'll take a timeboxed look into it